### PR TITLE
Bluetooth: shell: Add advertising options one-time and directed privacy

### PR DESCRIPTION
**Pull request by joerchan**
Add advertising option for undirected one-time advertising, and
directed advertising using identity, and peer direct is RPA option.

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/shell/bt.c [hunk 1]
```diff
@@ -2401,11 +2411,13 @@ SHELL_STATIC_SUBCMD_SET_CREATE(bt_cmds,
 #if defined(CONFIG_BT_BROADCASTER)
 	SHELL_CMD_ARG(advertise, NULL,
 		      "<type: off, on, scan, nconn> [mode: discov, non_discov] "
-		      "[whitelist: wl, wl-scan, wl-conn] [identity] [no-name]",
-		      cmd_advertise, 2, 4),
+		      "[whitelist: wl, wl-scan, wl-conn] [identity] [no-name]"
```
Comment (carlescufi) [inline subsys/bluetooth/shell/bt.c:2414]: missing a space at the end of the string?

Comment (joerchan) [reply subsys/bluetooth/shell/bt.c]: Yup, fixed.

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@jhedberg any objections to merging this?

**Review by jhedberg — APPROVED** [review]
> @jhedberg any objections to merging this?

@carlescufi nope, looks good

**Pull request merged**